shared/common-adapters/plain-input.desktop.js

```diff
@@ -170,7 +170,7 @@ class PlainInput extends React.PureComponent<InternalProps> {
 
   render = () => {
     const inputProps = this._getInputProps()
-    const css = `::-webkit-input-placeholder { color: rgba(0,0,0,.4); }
+    const css = `::-webkit-input-placeholder { color: ${this.props.placeholderColor || 'rgba(0,0,0,.4)'}; }
```

Comment:
kinda curious why we have to use css here instead of doing it inline? you can do WebkitInputPlaceholder i believe

Reply:
black_40

Reply:
It was a remnant of `Input.desktop`, I can't find anything about how to style pseudo-elements inline

Reply:
k we should still change to black_40